### DISCUSSION TITLE
刷新问题

### DISCUSSION THREAD

**Discussion by JonouZ** [electerm/electerm#3255] [Ideas]
![image](https://github.com/electerm/electerm/assets/56708898/7a725538-9eec-4674-8c4a-fa9983126d02)
为啥长时间没有操作，会需要刷新呢

**Comment by zxdong262** [comment]
可能网络波动导致断线，可以把设置里保活间隔设为0试试

**Reply by JonouZ** [reply]
设置的就是0 ，但是还是有这个问题，我设置 了时间也不行，

**Comment by finaljoy** [comment]
这个问题确实影响使用，因为刷新后不会记忆最后的工作目录

**Comment by Smalldy** [comment]
期间电脑休眠过吗，是否有线到wifi切换之类的操作
